internal/builder/builder.go

```diff
@@ -259,7 +258,7 @@ func (b *Builder) SetStack(stackConfig builder.StackConfig) {
 }
 
 // Save saves the builder
-func (b *Builder) Save(logger logging.Logger) error {
+func (b *Builder) Save(logger logging.Logger, version string) error {
```

Comment:
With the caveat that the architecture of `Builder` has been problematic and due for some refactor, I think at minimum to make this change acceptable we should change it from `..., version string)` to `..., creator CreatorMetadata)` so that it's up to the caller to provide the entire set of information for `CreatedBy`.